src/axom/primal/geometry/Plane.hpp

```diff
@@ -209,6 +222,19 @@ class Plane
   AXOM_HOST_DEVICE int getOrientation(const PointType& x,
                                       double TOL = 1.e-9) const;
 
+  /*!
+   * \brief Simple check for validity of a Plane
+   *
+   * Check that the normal is a unit vector (which also handles the zero
+   * vector case).
```

Comment:
See my comment above -- but I think we should revisit the notion that a plane must have a normalized unit vector to be valid.

Perhaps it should be valid if the normal is not the zero vector (perhaps up to some epsilon) ?

Reply:
I'm fine with the notion of a valid plane being one that has a non-zero vector and removing the requirement that it be a unit vector.

Reply:
We'd have to be careful in a few places within axom and update the RELEASE_NOTES, 
but I don't think `primal::Plane` is that widely used (at least internally)

Reply:
Are you suggesting we also change the default behavior to be `normalize = false`?

Reply:
> Are you suggesting we also change the default behavior to be normalize = false?

Yes. This changes the default behavior, but I don't see a good reason to require unit vectors by default (which involves extra computational work and might affect answers).

Reply:
We could make this change in a follow-up PR if you feel uneasy about too many changes in a single PR, or if the team wants to discuss this further.

Reply:
I changed the isValid check to only make sure the normal vector is not zero. However, I left the default behavior the same (it will still be normalized by default). I'll change that in a future merge request.